Unix/engine/engine.c

```diff
@@ -0,0 +1,119 @@
+/*
+**==============================================================================
+**
+** Copyright (c) Microsoft Corporation. All rights reserved. See file LICENSE
+** for license information.
+**
+**==============================================================================
+*/
+
+#include <server/server.h>
+
+static Options s_opts;
+static ServerData s_data;
+
+int enginemain(int argc, const char* argv[])
+{
+    int pidfile = -1;
+    MI_Result result;
+
+    SetDefaults(&s_opts, &s_data, argv[0], OMI_ENGINE);
+
+    /* Get --destdir command-line option */
+    GetCommandLineDestDirOption(&argc, argv);
+
+    /* Extract configuration file options */
+    GetConfigFileOptions();
+
+    /* Extract command-line options a second time (to override) */
+    GetCommandLineOptions(&argc, argv);
+
+    /* Open the log file */
+    OpenLogFile();
+
+#if defined(CONFIG_POSIX)
```

Comment:
leftover from days of supporting Windows.  If we want to remove Windows support, we can go in and clean them up.